devops/scripts/publish_register_bundle.sh

```diff
@@ -65,25 +65,25 @@ while [ "$1" != "" ]; do
 done
 
 if [[ -n ${tre_url+x} ]]; then
-    if [[ -z ${access_token+x} ]]; then  
+    if [[ -z ${access_token+x} ]]; then
         echo -e "No Azure access token provided\n"
```

Comment:
The command in documentation won't work if the script is halted here, see https://github.com/microsoft/AzureTRE/compare/develop...tompaana/docs-fixes?expand=1